sklearn/manifold/t_sne.py

```diff
@@ -644,8 +645,10 @@ class TSNE(BaseEstimator):
         http://lvdmaaten.github.io/publications/papers/JMLR_2014.pdf
     """
 
+    @property
```

Comment:
The ``@property`` is just for deprecating attributes, not functions. That's why the tests fail, I think.